python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -23,114 +23,119 @@
-                # json string is massive (with 50,000 points: 16mb uncompressed, 2.5mb compressed).
-                # A quick test using delimented data brings this down to ~1mb compressed.
-                map_result = cache.get(MINE_MAP_CACHE)
-                last_modified = cache.get(MINE_MAP_CACHE + '_LAST_MODIFIED')
-                if not map_result:
-                    records = MineMapViewLocation.query.filter(MineMapViewLocation.latitude != None)
-                    last_modified = datetime.utcnow()
-
-                    # jsonify then store in cache
-                    map_result = json.dumps(
-                        {
-                            'mines': list((map(lambda x: x.json_for_map(), records)))
-                        },
-                        separators=(',', ':'))
-                    cache.set(MINE_MAP_CACHE, map_result, timeout=TIMEOUT_12_HOURS)
-                    cache.set(
-                        MINE_MAP_CACHE + '_LAST_MODIFIED', last_modified, timeout=TIMEOUT_12_HOURS)
-
-                # It's more efficient to store the json to avoid re-initializing all of the objects
-                # and jsonifying on every request, so a flask response is returned to prevent
-                # flask_restplus from jsonifying the data again, which would mangle the json.
-                response = make_response(map_result)
-                response.headers['content-type'] = 'application/json'
-
-                # While we're at it, let's set a last modified date and have flask return not modified
-                # if it hasn't so the client doesn't download it again unless needed.
-                response.last_modified = last_modified
-                response.make_conditional(request)
-
-                return response
-
-            paginated_mine_query, pagination_details = self.apply_filter_and_search(request.args)
-            mines = paginated_mine_query.all()
-            return {
-                'mines': list(map(lambda x: x.json_for_list(), mines)),
-                'current_page': pagination_details.page_number,
-                'total_pages': pagination_details.num_pages,
-                'items_per_page': pagination_details.page_size,
-                'total': pagination_details.total_results,
-            }
+    def get(self):
+
+        paginated_mine_query, pagination_details = self.apply_filter_and_search(request.args)
+        mines = paginated_mine_query.all()
+        return {
+            'mines': mines,
+            'current_page': pagination_details.page_number,
+            'total_pages': pagination_details.num_pages,
+            'items_per_page': pagination_details.page_size,
+            'total': pagination_details.total_results,
+        }
+
+    @api.expect(parser)
+    @api.doc(
+        description=
+        'This endpoint creates a new mine with the information provided and returns it. If an error occurs the appropriate response is returned.'
```

Comment:
I'd like us to avoid using "This endpoint" at the start of every description. An active, user-action-focused voice is much more direct and concise. "Create a new mine."

The "with the information provided" is implied by the nature of a REST API, imo. "and returns it" should be obvious from the rest of the documentation for the endpoint.

And I think it's reasonable for someone to expect an error to be returned if an error occurs. Adding that to the description is unnecessarily wordy